crates/gpui_macros/Cargo.toml

```diff
@@ -16,3 +16,6 @@ proc-macro = true
 proc-macro2 = "1.0.66"
 quote = "1.0.9"
 syn = { version = "1.0.72", features = ["full"] }
+
+[dev-dependencies]
+gpui = { workspace = true, features = ["test-support"] }
```

Comment:
`gpui` depends on `gpui_macros`, so I'm not sure what the implications are of adding a dev dependency here.

I think we may just want to disable doctests for `gpui_macros`:

```diff
  [lib]
  path = "src/gpui_macros.rs"
  proc-macro = true
+ doctest = false
```

Reply:
We should let it run if we think it is doctest because doctest makes sure that examples within your documentation are up to date and working. so it is not a good idea to disable doctests in any modules.

I think the better idea is to remove the three back-quotes(```) from comments of source code and make sure it is not a doctest anymore.

Reply:
We don't want to remove the code block, as that shows up in the editor.

We already have prior art for disabling doctests in some of our other crates, so I disabled them for `gpui_macros` in #13015.